dynawo/sources/Models/Modelica/Dynawo/Electrical/HVDC/SimplifiedHVDCACEmulation/SimplifiedHVDCACEmulation.mo

```diff
@@ -0,0 +1,133 @@
+within Dynawo.Electrical.HVDC.SimplifiedHVDCACEmulation;
+
+/*
+* Copyright (c) 2015-2020, RTE (http://www.rte-france.com)
+* See AUTHORS.txt
+* All rights reserved.
+* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, you can obtain one at http://mozilla.org/MPL/2.0/.
+* SPDX-License-Identifier: MPL-2.0
+*
+* This file is part of Dynawo, an hybrid C++/Modelica open source time domain simulation tool for power systems.
+*/
+
+model SimplifiedHVDCACEmulation "Simplified model of HVDC link with AC Emulation"
+
+/*
+  Equivalent circuit and conventions:
+
+               I1                  I2
+   (terminal1) -->-------HVDC-------<-- (terminal2)
+
+*/
+
+  import Modelica;
+  import Dynawo.Connectors;
+  import Dynawo.Electrical.Controls.Basics.SwitchOff;
+
+  extends SwitchOff.SwitchOffLine;
+  extends AdditionalIcons.Line;
+
+  parameter Types.ComplexVoltagePu u10Pu  "Start value of complex voltage at terminal 1 in p.u (base UNom)";
```

Comment:
These start values should be protected and appear after public parameters and variables

Reply:
Done.